Objective-C/Tests/DatabaseTest.m

```diff
@@ -2640,6 +2640,176 @@ - (void) testCollection {
     
 }
 
+#pragma mark - GET Document
```

Comment:
As database is not the data store anymore, I think it is better to have all of the documents test else where such as in DocumentTest or a new CollectionDocumentTest.